### PR TITLE
Add example for Organization Access

### PR DESCRIPTION
**Pull request by Xorima**
Extends the examples to cover adding org access

### CODE REVIEW

File: website/docs/r/team.html.markdown [hunk 1]
```diff
@@ -21,6 +21,18 @@ resource "tfe_team" "test" {
...
 
+Organization Permission usage:
+
+```hcl
+resource "tfe_team" "test" {
+  name         = "my-team-name"
+  organization = "my-org-name"
+    organization_access {
+      manage_vcs_settings = true
+    }
```
Comment (brandonc) [inline website/docs/r/team.html.markdown:32]: I think this just need a de-intent

```suggestion
  organization_access {
    manage_vcs_settings = true
  }
```

**Review by brandonc — APPROVED** [review]
Thanks for improving the docs!

**Comment by Xorima** [comment]
One small thing to note here, we are unable to directly permission the private registry modules & providers as the upstream golang module doesn't support this from what I can tell

**Comment by Xorima** [comment]
So I no longer have access as I just left my previous employer to send you a screenshot, but if on a fully licensed TFC account you go into a team and look at organization permissions in the UI you will see as well something like:
- Manage Registry
  - Manage Modules
  - Manage Providers

This all relates to the private TF registry you get in TFC, it was frustrating that I could not perm this via code itself and has to be manually done on relevant teams.

**Pull request merged**